.luacheckrc

```diff
@@ -0,0 +1,4 @@
+ignore = {"213/_.*"} -- unused Loop Variables beginning with underscore
+files["openwisp-config/tests"] = {
+    ignore = {"11./Test.*"} -- globals relating to defining unittests
+}
```

Comment:
final new line is missing, please configure your editor to always have new lines.

Reply:
Will do